testdata/ansible/memcached-operator/config/manager/manager.yaml

```diff
@@ -31,4 +31,16 @@ spec:
             - name: ANSIBLE_GATHERING
               value: explicit
           image: controller:latest
+          livenessProbe:
+            httpGet:
+              path: /readyz
+              port: 6789
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: 6789
+            initialDelaySeconds: 5
+            periodSeconds: 10
```

Comment:
livenessProbe using /readyz endpoint and readinessProbe using /healthz endpoint. Should be the other way around I guess.